CMakeLists.txt

```diff
@@ -202,7 +202,7 @@ endif ()
 # Everything should be valid from here onwards.
 if (OPENCMISS_SETUP_TYPE STREQUAL "standard" AND NOT OPENCMISS_INDEPENDENT)
     set(OPENCMISS_MANAGE_SOURCE_DIR "${OPENCMISS_ROOT}/src/manage")
-    set(OPENCMISS_MANAGE_BINARY_DIR "${OPENCMISS_ROOT}/build/manage${SINGLE_ARCHITECUTRE_BUILD_TYPE}")
+    set(OPENCMISS_MANAGE_BINARY_DIR "${OPENCMISS_ROOT}/build/manage/${CMAKE_HOST_SYSTEM_NAME}${SINGLE_ARCHITECUTRE_BUILD_TYPE}")
```

Comment:
Adding this prefix should be optional with the default option being not to have it.

Reply:
Why? It is safer to have it. It makes documenting where to go in manage much more difficult because it is there. As it there it means that files are separated so that when developers add different configs then they don't have to blow away what they have. For the amount of time people need to go to this directory the overhead of an additional tab button press is completely trivial. What is there that shouldn't be is release which is completely un-necessary and so if it is OK for that to be there for as long as it has been then this should not be a problem. For developers multi-config should be the default. Users wouldn't even go into this directory.